core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -129,10 +130,11 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
     line-height: 0.6;
     padding-left: 4px;
     text-overflow: clip;
+    text-align: center;
```

Comment:
hi @naveenshukla , thanks for working on this! I think your ideas for removing the social media stuff, for the reasons you give, are very smart. I would ask, as Sean mentions, that we continue to align the AUTHOR PROFILES to the left (by the apple, which is meant as sort of a "teacher" icon). I think the next-to-last mock screenshot you posted looked just right, minus the social media stuff. Thanks!